server/subsonic/responses/responses.go

```diff
@@ -231,7 +231,8 @@ type AlbumID3 struct {
 	IsCompilation       bool       `xml:"isCompilation,attr"    json:"isCompilation"`
 	SortName            string     `xml:"sortName,attr"         json:"sortName"`
 	DiscTitles          DiscTitles `xml:"discTitles"            json:"discTitles"`
-	OriginalReleaseDate ItemDate   `xml:"originalReleaseDate"   json:"originalReleaseDate"`
+	OriginalReleaseDate ItemDate   `xml:"originalReleaseDate"   json:"originalReleaseDate,omitempty"`
+	ReleaseDate         ItemDate   `xml:"releaseDate" json:"releaseDate,omitempty"`
```

Comment:
I prefer to not have `omitempty` in these fields, to keep it consistent with other OS fields. Also because this will never be the case (them being empty and omitted)